test/hook.afterPackageCreateDeploymentArtifacts.test.js

```diff
@@ -74,4 +74,149 @@ describe('Serverless warmup plugin after:deploy:deploy hook', () => {
 
     expect(fs.remove).not.toHaveBeenCalled()
   })
+
+  it('Should have default global package property', async () => {
```

Comment:
It's not really global.
I'd rename to `Should package only the lambda handler by default`

Reply:
renamed